src/custom/api/gnosisProtocol/api.ts

```diff
@@ -23,7 +23,7 @@ import QuoteError, {
   mapOperatorErrorToQuoteError,
 } from 'api/gnosisProtocol/errors/QuoteError'
 import { toErc20Address, toNativeBuyAddress } from 'utils/tokens'
-import { FeeQuoteParams, PriceInformation, PriceQuoteParams, SimpleGetQuoteResponse } from 'utils/price'
+import { LegacyFeeQuoteParams as FeeQuoteParams, LegacyPriceQuoteParams as PriceQuoteParams } from './legacy/types'
```

Comment:
in the coming PR this file will mostly be removed as these methods are in the SDK